gast_to_code.py

```diff
@@ -84,6 +84,8 @@ def gast_router(gast, out_lang):
         return '"' + gast["value"] + '"'
     elif gast["type"] == "bool":
         return out["bool"][out_lang](gast)
+    elif gast["type"] == "none":
+        return "None" if out_lang == "py" else "null"
```

Comment:
Are we making a distinction between null and undefined in JS?

Reply:
+1. If not, I would add a comment stating that we are arbitrarily choosing null.

Reply:
We have decided to not worry about undefined for now and could possibly add in later. I will add a comment.